apps/extension/src/ui/domains/Settings/DatabaseErrorAlert.tsx

```diff
@@ -0,0 +1,104 @@
+import { errorsStore } from "@core/domains/app"
+import { Card } from "@talisman/components/Card"
+import { AlertCircleIcon, DatabaseIcon } from "@talismn/icons"
+import { AnalyticsPage } from "@ui/api/analytics"
+import { useErrorsStoreValue } from "@ui/hooks/useErrors"
+import { useRuntimeReload } from "@ui/hooks/useRuntimeReload"
+import { useCallback, useMemo } from "react"
+import { Trans, useTranslation } from "react-i18next"
+import { Button } from "talisman-ui"
+
+const ANALYTICS_PAGES: Record<"popup" | "fullscreen", AnalyticsPage> = {
+  popup: {
+    container: "Popup",
+    feature: "Portfolio",
+    featureVersion: 1,
+    page: "Database Unavailable",
+  },
+  fullscreen: {
+    container: "Fullscreen",
+    feature: "Portfolio",
+    featureVersion: 1,
+    page: "Database Unavailable",
+  },
+}
+
+export type Props = {
+  container: keyof typeof ANALYTICS_PAGES
+}
+
+export const DatabaseErrorAlert = ({ container }: Props) => {
```

Comment:
This is awesome, as an alternative you could derive the `container` from the current location rather than requiring it as a prop. Not too important though